gym/envs/registration.py

```diff
@@ -1,10 +1,10 @@
-import re
-import sys
+import contextlib
```

Comment:
Are these import formatting changes necessary to pass CI now?

Reply:
Same as https://github.com/openai/gym/pull/2535#discussion_r776417773